tests/test_service.py

```diff
@@ -830,3 +830,183 @@ def testGetAlertWithMultipleTriggers(self):
+            self.assertTrue(isinstance(trigger, Trigger))
+            call_args = tuple(mock_trigger_post.call_args)
+            uri_path = os.path.join(endpoint, "alerts/{}/triggers".format(child_alert.id))
+            self.assertIn((uri_path,), call_args)
+
+        with mock.patch('requests.Session.delete', return_value=MockResponse("", 200)) as mock_delete:
+            self.argus.alerts.del_trigger_from_child_alert(child_alert.id, trigger.id)
+            call_args = tuple(mock_trigger_post.call_args)
+            uri_path = os.path.join(endpoint, "alerts/{}/triggers".format(child_alert.id))
+            self.assertIn((uri_path,), call_args)
+
+    def testDeleteNotification(self):
+        with mock.patch('requests.Session.post', return_value=MockResponse(json.dumps(compalert_D), 200)) as mock_add_comp_alert:
+            alert = Alert.from_dict(compalert_D)
+            self.assertTrue(isinstance(alert, Alert))
+            delattr(alert, "id")
+            comp_alert = self.argus.alerts.create_composite_alert(alert)
+            self.assertTrue(isinstance(comp_alert, Alert))
+            call_args = mock_add_comp_alert.call_args
+            uri_path = os.path.join(endpoint, "alerts")
+            self.assertIn((uri_path,), call_args)
+
+        with mock.patch('requests.Session.post', return_value=MockResponse(json.dumps(compAlert_notification), 200)) as mock_notification:
+            notification = self.argus.alerts.add_notification_to_composite_alert(comp_alert, Notification.from_dict(compAlert_notification))
+            self.assertTrue(isinstance(notification, Notification))
+            call_args = mock_notification.call_args
+            uri_path = os.path.join(endpoint, "alerts/{}/notifications".format(comp_alert.id))
+            self.assertIn((uri_path,), call_args)
+
+        with mock.patch('requests.Session.delete', return_value=MockResponse("", 200)) as mock_delete:
+            self.argus.alerts.del_notification_from_composite_alert(comp_alert, notification.id)
+            call_args = tuple(mock_delete.call_args)
+            uri_path = os.path.join(endpoint, "alerts/{}/notifications/{}".format(comp_alert.id, notification.id))
+            self.assertIn((uri_path,), call_args)
+
+    def testGetCompAlertChildrenInfo(self):
+        with mock.patch('requests.Session.get', return_value=MockResponse(json.dumps([childAlert_1, childAlert_2]), 200)) as mock_get:
+            res = self.argus.alerts.get_composite_alert_children_info(compAlertID)
+            if res:
+                for obj in res:
+                    self.assertTrue(isinstance(obj, Alert))
+            call_args = tuple(mock_get.call_args)
+            uri_path = os.path.join(endpoint, "alerts/{}/children/info".format(compAlertID))
+            self.assertIn((uri_path,), call_args)
+
+    def testUpdateCompAlert(self):
+        with mock.patch('requests.Session.post', return_value=MockResponse(json.dumps(compalert_D), 200)) as mock_add_comp_alert:
```

Comment:
Should 'mock_add_comp_alert' have 'update' instead?

Reply:
First we add and then we update.  If you look at the second section i do call mock_update_comp_alert